src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -603,13 +591,13 @@ private void commitContractCache(Deposit deposit) {
   }
 
   private void commitStorageCache(Deposit deposit) {
-    storageCache.forEach((key, value) -> {
+    storageCache.forEach((address, storage) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Specify a type for: 'address', 'storage' ([squid:S2211](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2211))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6pnSgN4uCclo1c3&open=AWbJU6pnSgN4uCclo1c3)

[ISSUE_KEY]: AWbJU6pnSgN4uCclo1c3